transcripts/share-apis/code-browse/search.json

```diff
@@ -4,7 +4,8 @@
       "avatarUrl": null,
       "handle": "test",
       "name": null,
-      "tag": "User"
+      "tag": "User",
+      "userId": "U-<UUID>"
```

Comment:
Here's an example response, except the transcript runner filters out UUIDs since they're random, but you get the idea :)